content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -209,6 +209,8 @@ If you prefer to manually correlate your traces with your logs, you can add corr
 
 **Note:** If you are not using a [Datadog Log Integration][7] to parse your logs, custom log parsing rules must parse `dd.trace_id` and `dd.span_id` as strings. For information, see the [FAQ on this topic][10].
 
+**Note**: If you are using Serilog, Nlog or log4net *through* ILogger, see the Microsoft.Extensions.Logging section to configure these properties via `BeginScope()`.
```

Comment:
```suggestion
**Note**: If you are using Serilog, Nlog or log4net with ILogger, see the Microsoft.Extensions.Logging section to configure these properties using `BeginScope()`.
```

Reply:
I think through is the better word, but I removed the bold. You call the ILogger APIs and that calls the Serilog/NLog/Etc. However, if you feel strongly about it, I will to your expertise.

Reply:
No problem, that works.